client/packages/system/src/Stock/api/hooks/useStockLine.ts

```diff
@@ -15,76 +16,185 @@ export interface DraftStockLine extends StockLineRowFragment {
   inventoryAdjustmentReason: InventoryAdjustmentReasonRowFragment | null;
 }
 
-export function useStockLine() {
-  const [draft, setDraft] = useState<DraftStockLine>({
-    __typename: 'StockLineNode',
-    id: '',
-    itemId: '',
-    onHold: false,
-    packSize: 0,
-    sellPricePerPack: 0,
-    costPricePerPack: 0,
-    totalNumberOfPacks: 0,
-    availableNumberOfPacks: 0,
-    storeId: '',
-    item: {
-      __typename: 'ItemNode',
-      code: '',
-      name: '',
-    },
-    inventoryAdjustmentReason: null,
-  });
+const defaultDraftStockLine = {
```

Comment:
```suggestion
const defaultDraftStockLine: DraftStockLine = {
```
nice to have type safety in the future

Reply:
79aae858d